HotelBooking.API/Controllers/BookingController.cs

```diff
@@ -41,15 +45,30 @@ public async Task<IActionResult> CreateBookingAsync([FromForm] BookingVM model)
                 logger.LogError("Invalid model sent from client.");
                 return BadRequest("Invalid model object");
             }
-            var res = await bookingService.AddBookingAsync(model);
+            var id = currentUser.UserId;
```

Comment:
Do we need to declare id var in this case?

Reply:
Do you mean the user can create a booking when they don't have an account?
I think they must register an account before they book a hotel.